packages/sil-governance/src/routes/create-proposal/components/FormOptions/DistributeFunds/RecipientsMembers.tsx

```diff
@@ -0,0 +1,57 @@
+import { Collapse, Table, TableBody, TableCell, TableHead, TableRow } from "@material-ui/core";
+import { Block, Typography } from "medulas-react-components";
+import React, { useMemo, useState } from "react";
+
+import { Recipient } from ".";
+
+interface Props {
+  readonly recipients: readonly Recipient[];
+}
+
+const RecipientsMembers = ({ recipients }: Props): JSX.Element => {
+  const [expanded, setExpanded] = useState(false);
+  const recipientsField = expanded ? "Hide recipients" : "Show recipients";
+
+  const onClick = (): void => {
+    setExpanded(prevExpanded => !prevExpanded);
+  };
+
+  const MembersTable = useMemo(
```

Comment:
We do not need to use `useMemo` here. Simply move this code into separate component with `recipients` as the prop.